src/test/java/nextstep/security/oauth2/OAuth2AuthorizationRequestRedirectFilterTest.java

```diff
@@ -0,0 +1,48 @@
+package nextstep.security.oauth2;
+
+import nextstep.app.domain.Member;
+import nextstep.security.SecurityApplicationTest;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+
+import java.util.Set;
+
+@AutoConfigureMockMvc
+@SecurityApplicationTest
+class OAuth2AuthorizationRequestRedirectFilterTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Test
+    void githubRedirectTest() throws Exception {
+        String requestUri = "/oauth2/authorization/github";
+        String expectedRedirectUri = "https://github.com/login/oauth/authorize" +
+                "?client_id= ..." +
```

Comment:
아래 테스트 코드가 실패하고 있네요 😢 
```suggestion
                "?client_id=..." +
```

Reply:
아, 시크릿을 제거하고 테스트 확인을 못했네요 ㅜㅜ 감사합니다 !!